metaworld/policies/sawyer_reach_v2_policy.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+from metaworld.policies.action import Action
+from metaworld.policies.policy import Policy, assert_fully_parsed, move
+
+
+class SawyerReachV2Policy(Policy):
+
+    @staticmethod
+    @assert_fully_parsed
+    def parse_obs(obs):
+        return {
+            'hand_xyz': obs[:3],
+            'puck_xyz': obs[3:-3],
+            'goal_vec': obs[-3:]
+        }
+
+    def get_action(self, obs):
```

Comment:
can you make everything but `get_action` on all of these classes private?

Reply:
Yeah I'll open a separate PR for that and do it for all the policies at once